crates/nu-command/src/filesystem/start.rs

```diff
@@ -34,45 +35,68 @@ impl Command for Start {
         engine_state: &EngineState,
         stack: &mut Stack,
         call: &Call,
-        input: PipelineData,
+        _input: PipelineData,
     ) -> Result<PipelineData, ShellError> {
-        let path = call.opt::<Spanned<String>>(engine_state, stack, 0)?;
-
-        let path = {
-            if let Some(path_val) = path {
-                Some(Spanned {
-                    item: nu_utils::strip_ansi_string_unlikely(path_val.item),
-                    span: path_val.span,
-                })
-            } else {
-                path
-            }
+        let path = call.req::<Spanned<String>>(engine_state, stack, 0)?;
+        let path = Spanned {
+            item: nu_utils::strip_ansi_string_unlikely(path.item),
+            span: path.span,
         };
-
-        let path = if let Some(path) = path {
-            path
+        let path_no_whitespace = &path.item.trim_end_matches(|x| matches!(x, '\x09'..='\x0d'));
+        // only check if file exists in current current directory
+        let file_path = Path::new(path_no_whitespace);
+        if file_path.exists() {
+            open::that(path_no_whitespace)?;
+        } else if file_path.starts_with("https://") || file_path.starts_with("https://") {
```

Comment:
starts_with https or starts_with https?

Reply:
uuupps, lol. Fixed! I meant `http://` :)

Reply:
figured that's what it was, LOL.